Frameworks/Foundation/NSUserDefaults.mm

```diff
@@ -245,6 +249,12 @@ - (void)setObject:(id)value forKey:(NSString*)key {
     CFPreferencesSetAppValue(static_cast<CFStringRef>(key), valueCopy, kCFPreferencesCurrentApplication);
```

Comment:
On the contrary, the documentation for  CFPreferencesSetAppValue states "You must call the CFPreferencesAppSynchronize function in order for your changes to be saved to permanent storage." - Therefore this automatic synchronize logic only needs to exist in NSUserDefaults.

---
In reply to: [69979853](https://github.com/Microsoft/WinObjC/pull/612#discussion_r69979853) [](ancestors = 69979853)